scalanet/src/io/iohk/scalanet/peergroup/kademlia/KRouter.scala

```diff
@@ -146,7 +194,6 @@ class KRouter[A](val config: Config[A], val network: KNetwork[A])(
             info(s"Query to node $knownNode failed due to exception $t.")
             Seq.empty
         }
-
     }
 
     def loop(closestNodes: Seq[BitVector]): Future[Seq[NodeRecord[A]]] = {
```

Comment:
It seems that this method did not fix a https://github.com/input-output-hk/scalanet/pull/43#discussion_r314784477 from the previous PR